ENIGMAsystem/SHELL/Universal_System/Resources/fontstruct.cpp

```diff
@@ -286,48 +269,48 @@ int font_add_sprite(int spr, uint32_t first, bool prop, int sep)
 
   unsigned char gcount = sspr->subcount;
   int idfont = enigma::font_new(first, gcount);
-  enigma::font *font = enigma::fontstructarray[idfont];
+  enigma::font *font = &enigma::fonts[idfont];
   if (!enigma::font_pack(font, spr, gcount, prop, sep)) return -1;
   return idfont;
 }
 
 float font_get_glyph_texture_left(int fnt, uint32_t character) {
-  const enigma::fontglyph glyph = enigma::findGlyph(enigma::fontstructarray[fnt], character);
+  const enigma::fontglyph glyph = enigma::findGlyph(enigma::fonts[fnt], character);
   return glyph.tx;
 }
 
 float font_get_glyph_texture_top(int fnt, uint32_t character) {
-  const enigma::fontglyph glyph = enigma::findGlyph(enigma::fontstructarray[fnt], character);
+  const enigma::fontglyph glyph = enigma::findGlyph(enigma::fonts[fnt], character);
   return glyph.ty;
 }
 
 float font_get_glyph_texture_right(int fnt, uint32_t character) {
-  const enigma::fontglyph glyph = enigma::findGlyph(enigma::fontstructarray[fnt], character);
+  const enigma::fontglyph glyph = enigma::findGlyph(enigma::fonts[fnt], character);
   return glyph.tx2;
 }
 
 float font_get_glyph_texture_bottom(int fnt, uint32_t character) {
-  const enigma::fontglyph glyph = enigma::findGlyph(enigma::fontstructarray[fnt], character);
+  const enigma::fontglyph glyph = enigma::findGlyph(enigma::fonts[fnt], character);
   return glyph.ty2;
 }
 
 float font_get_glyph_left(int fnt, uint32_t character) {
-  const enigma::fontglyph glyph = enigma::findGlyph(enigma::fontstructarray[fnt], character);
+  const enigma::fontglyph glyph = enigma::findGlyph(enigma::fonts[fnt], character);
   return glyph.x;
 }
 
 float font_get_glyph_top(int fnt, uint32_t character) {
-  const enigma::fontglyph glyph = enigma::findGlyph(enigma::fontstructarray[fnt], character);
+  const enigma::fontglyph glyph = enigma::findGlyph(enigma::fonts[fnt], character);
   return glyph.y;
 }
 
 float font_get_glyph_right(int fnt, uint32_t character) {
-  const enigma::fontglyph glyph = enigma::findGlyph(enigma::fontstructarray[fnt], character);
+  const enigma::fontglyph glyph = enigma::findGlyph(enigma::fonts[fnt], character);
   return glyph.x2;
 }
 
 float font_get_glyph_bottom(int fnt, uint32_t character) {
-  const enigma::fontglyph glyph = enigma::findGlyph(enigma::fontstructarray[fnt], character);
+  const enigma::fontglyph glyph = enigma::findGlyph(enigma::fonts[fnt], character);
```

Comment:
Since we're renaming this everywhere, maybe it's best to just call it `sprite_fonts`.